### PR TITLE
[EPMRPP-45445] Fix cleaning log attachments

### PR DESCRIPTION
**Pull request by ihar-kahadouski**

### CODE REVIEW

**Comment by KostyaSha** [comment]
# [Codecov](https://codecov.io/gh/reportportal/service-api/pull/1118?src=pr&el=h1) Report
> Merging [#1118](https://codecov.io/gh/reportportal/service-api/pull/1118?src=pr&el=desc) into [develop](https://codecov.io/gh/reportportal/service-api/commit/2b8943e7d21d57d8e2accdd494d339e6308007f6&el=desc) will **decrease** coverage by `0.00%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/reportportal/service-api/pull/1118?src=pr&el=tree)

```diff
@@              Coverage Diff              @@
##             develop    #1118      +/-   ##
=============================================
- Coverage      72.62%   72.61%   -0.01%     
+ Complexity      2154     2152       -2     
=============================================
  Files            407      407              
  Lines          10105    10102       -3     
  Branches         460      460              
=============================================
- Hits            7339     7336       -3     
  Misses          2446     2446              
  Partials         320      320              
```



------

[Continue to review full report at Codecov](https://codecov.io/gh/reportportal/service-api/pull/1118?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/reportportal/service-api/pull/1118?src=pr&el=footer). Last update [2b8943e...3ac61d3](https://codecov.io/gh/reportportal/service-api/pull/1118?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

File: src/main/java/com/epam/ta/reportportal/job/CleanLogsJob.java [hunk 1]
```diff
@@ -57,8 +55,7 @@
 
 	private static final Logger LOGGER = LoggerFactory.getLogger(CleanLogsJob.class);
 	public static final int DEFAULT_THREAD_COUNT = 5;
-	public static final long JOB_EXECUTION_TIMEOUT = 1L;
-	public static final Duration MIN_DELAY = Duration.ofDays(KeepLogsDelay.TWO_WEEKS.getDays() - 1);
+	//	public static final Duration MIN_DELAY = Duration.ofDays(KeepLogsDelay.TWO_WEEKS.getDays() - 1);
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/job/CleanLogsJob.java]: ?

File: src/main/java/com/epam/ta/reportportal/job/CleanLogsJob.java [hunk 2]
```diff
@@ -57,8 +55,7 @@
 
 	private static final Logger LOGGER = LoggerFactory.getLogger(CleanLogsJob.class);
 	public static final int DEFAULT_THREAD_COUNT = 5;
-	public static final long JOB_EXECUTION_TIMEOUT = 1L;
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/job/CleanLogsJob.java]: why is it removed?

File: src/test/java/com/epam/ta/reportportal/job/LaunchCleanerServiceImplTest.java [hunk 3]
```diff
@@ -57,13 +56,13 @@ void runTest() {
...
 
-		int removedLaunchesCount = 3;
-		when(launchRepository.deleteLaunchesByProjectIdModifiedBefore(any(), any())).thenReturn(removedLaunchesCount);
-
-		launchCleanerService.cleanOutdatedLaunches(project, period, launchesRemoved);
-
-		assertEquals(removedLaunchesCount, launchesRemoved.get());
-		verify(activityRepository, times(1)).deleteModifiedLaterAgo(project.getId(), period);
-		verify(launchRepository, times(1)).deleteLaunchesByProjectIdModifiedBefore(eq(project.getId()), any(LocalDateTime.class));
+//		int removedLaunchesCount = 3;
```
Comment (pbortnik) [inline src/test/java/com/epam/ta/reportportal/job/LaunchCleanerServiceImplTest.java]: ?

Comment (pbortnik) [inline src/test/java/com/epam/ta/reportportal/job/LaunchCleanerServiceImplTest.java]: remove commented code or fix test

File: src/main/java/com/epam/ta/reportportal/job/CleanLogsJob.java (hunk 2 above)
Comment (ihar-kahadouski) [reply src/main/java/com/epam/ta/reportportal/job/CleanLogsJob.java]: Not used anywhere

File: src/test/java/com/epam/ta/reportportal/job/LaunchCleanerServiceImplTest.java (hunk 3 above)
Comment (ihar-kahadouski) [reply src/test/java/com/epam/ta/reportportal/job/LaunchCleanerServiceImplTest.java]: done

File: src/main/java/com/epam/ta/reportportal/job/CleanLogsJob.java (hunk 1 above)
Comment (ihar-kahadouski) [reply src/main/java/com/epam/ta/reportportal/job/CleanLogsJob.java]: removed

**Pull request merged**